test/TestApps/PushNotificationsDemoApp/main.cpp

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#include "pch.h"
+#include <wil/win32_helpers.h>
+#include <iostream>
+#include <time.h>
+#include <winrt/Windows.ApplicationModel.Background.h> // we need this for BackgroundTask APIs
+
+#define SIZE 26
+
+using namespace winrt::Microsoft::Windows::AppLifecycle;
+using namespace winrt::Microsoft::Windows::PushNotifications;
+using namespace winrt;
+using namespace winrt::Windows::Storage;
+using namespace winrt::Windows::Storage::Streams;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::ApplicationModel::Activation;
+using namespace winrt::Windows::ApplicationModel::Background; // BackgroundTask APIs
+
+time_t ltime;
+char buf[SIZE];
+
+void RequestChannel()
+{
+    // Register the AAD RemoteIdentifier for the App to receive Push
+    auto channelOperation = PushNotificationManager::CreateChannelAsync(
+        winrt::guid("ccd2ae3f-764f-4ae3-be45-9804761b28b2"));
```

Comment:
Please explain where this guid came from?

Reply:
Sure. I have used the instructions https://review.docs.microsoft.com/en-us/windows/uwp/design/shell/tiles-and-notifications/set-up-aad-app-registration-for-wns?branch=niparent%2Freunion-wns-registration
 which is the process all developers would follow when requesting for channels from WNS

Reply:
Yes